packages/dashboard/src/components/template/_indices.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import { observer } from 'mobx-react-lite';
+import { Indice } from './elements';
+
+const Indices: React.FC<TIndicesProps> = ({ indice_data }) => {
+    return (
+        <div className='dw-template__indices'>
+            <div>
+                {indice_data.map((item, idx) => (
+                    <div key={idx} className='dw-template__indices-container'>
```

Comment:
Can we use something else other than `idx` for `key`? Is any prop unique that could be used?

- https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Not really. This is the content of the page that the developer create an array for it. no code, no id, no unique key..